tests/test_multiaddr.py

```diff
@@ -250,7 +253,7 @@ def test_get_value_too_many_fields_protocol(monkeypatch):
     that the constructor specifies is ignored by the test.
     """
     monkeypatch.setattr("multiaddr.multiaddr.Multiaddr.__str__",
-                        lambda ignore: '/udp/1234/5678')
```

Comment:
(This was part of #36, not this PR.)

Since I added `from __future__ import unicode_literals` to the top of the test file, *all* strings are Unicode-based in both Python 2 & 3 unless explicitly marked as `b''`.
So while using the `str` function will do nothing on Python 3, it *will* convert the Unicode literal `u'/udp/1234/5678'` to `bytes` (aka `str`, the native string type) on Python 2. This is required since `__str__` always needs to return a `str` on both versions, but the definition of what `str` (obviously) differs between these platforms.